### PR TITLE
folderify v3.0.6 (switching from Python to Rust)

### PR DESCRIPTION
**Pull request by lgarron**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

Note that this removes the use of `seticon` from https://github.com/Homebrew/homebrew-core/pull/87346 (which therefor doesn't need a workaround).

### CODE REVIEW

File: Formula/folderify.rb [hunk 1]
```diff
@@ -1,63 +1,28 @@
...
   head "https://github.com/lgarron/folderify.git", branch: "main"
 
-  bottle do
-    sha256 cellar: :any_skip_relocation, arm64_ventura:  "a2b1e1d23423c152acb05f34a26cd5e6c5480116bc63a2868cef9659f1a42837"
-    sha256 cellar: :any_skip_relocation, arm64_monterey: "c42d0a7a535288bd8479f5071e8a973e8027bcaa259437667b20c50a2c4b9285"
-    sha256 cellar: :any_skip_relocation, arm64_big_sur:  "5bb8a9116732629a9698c44f2f533560a325dcd7a1eadbfbc7473598e07889da"
-    sha256 cellar: :any_skip_relocation, ventura:        "d6e1ee0027d66edcb184e0f8653d7bcea684dd511881c5bab69876d053dff8a6"
-    sha256 cellar: :any_skip_relocation, monterey:       "d6e1ee0027d66edcb184e0f8653d7bcea684dd511881c5bab69876d053dff8a6"
-    sha256 cellar: :any_skip_relocation, big_sur:        "1a714d0ecddc788eae807a62b55dcd6f7d5d864a3b0da91ba3446dfb5d6b9337"
-  end
```
Comment (carlocab) [inline Formula/folderify.rb:18]: Leave the bottle block please.

Comment (lgarron) [reply Formula/folderify.rb:18]: Restored!

File: Formula/folderify.rb [hunk 2]
```diff
@@ -1,10 +1,9 @@
 class Folderify < Formula
-  include Language::Python::Virtualenv
-
   desc "Generate pixel-perfect macOS folder icons in the native style"
   homepage "https://github.com/lgarron/folderify"
-  url "https://files.pythonhosted.org/packages/4c/18/a4d6491e4f64cbff4821cefa9fec1cfcb3048e19fc806d7e9af876654b94/folderify-2.4.0.tar.gz"
-  sha256 "daf1f5c64d59528d61d5a223d9ad2ba8f0e10ffd0d9cc2286ccd65b7fa516c24"
+  url "https://github.com/lgarron/folderify.git",
+    tag:      "v3.0.3",
+    revision: "843a99e7e796340a2809ae900fd415e6602fb8d5"
```
[the remark this answers is not in the document]
Comment (lgarron) [reply Formula/folderify.rb]: Sorry, I think my editor keeps messing with the indentation.

If I understand correctly, you're suggesting to stick with an archive, for which I presume https://github.com/lgarron/folderify/archive/refs/tags/v3.0.3.tar.gz can be used?

**Review by carlocab — APPROVED** [review]
Thanks!

**Comment by lgarron** [comment]
@carlocab: Could I ask if there's any particular reason `brew upgrade folderify` wouldn't work?

I repeatedly tried the following commands earlier today in different orders, and it stayed stuck on folderify `v2.4.0`.

```
brew update
brew upgrade folderify
brew install folderify
brew uninstall folderify && brew install folderify
brew search folderify
```

It seems to me that that the cached API data never updated.

I eventually got it working, and I think the thing that did it was removing `~/Library/Caches/Homebrew/`. Now that it's been re-created, `brew update` seems to work properly again.

I'd file a bug, but unfortunately it's not reproducible for me on the same machine. However, someone just informed me that they ran into the same thing: https://github.com/lgarron/folderify/issues/69

**Pull request merged**